socket.c

```diff
@@ -78,6 +79,12 @@ bool cSatipSocket::Open(const int portP, const bool reuseP)
      ERROR_IF_FUNC(setsockopt(socketDescM, SOL_IP, IP_PKTINFO, &yes, sizeof(yes)) < 0,
                    "setsockopt(IP_PKTINFO)", Close(), return false);
 #endif // __FreeBSD__
+     // Tweak receive buffer size if requested
+     if (rcvBufSizeM > 0)
+     {
+       ERROR_IF_FUNC(setsockopt(socketDescM, SOL_SOCKET, SO_RCVBUF, &rcvBufSizeM, sizeof(rcvBufSizeM)) < 0,
```

Comment:
default indentation in if-blocks is 3 whitespaces

Reply:
Done.